packages/config/config/devices/0x0438/1_kanal_bryter.json

```diff
@@ -15,6 +15,17 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "LifeLine",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Button 1",
+			"maxNodes": 5
+		}
+	},
 	"paramInformation": {
 		"1": {
 			"label": "Factory Reset",
```

Comment:
Remove the unnecessary description per the style guide. Please take a look at the other files and carry through.

Reply:
What is the unnecessary description you are talking about here @blhoward2 ?